dts/msm8974/apq8074-samsung-mondrianwifi.dts

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+#include <lk2nd.h>
+
+/ {
+	qcom,msm-id = <0xb801ff01 0x0c 0x00>;
+	compatible = "qcom,apq8074-cdp", "qcom,apq8074", "lk2nd,device";
+
+		model = "mondrianwifi";
+		compatible = "mondrianwifi", "qcom,apq8074", "lk2nd,device";
+
+
```

Comment:
Sorry, but this looks nonsensical to me. Please look at the nearby samsung files for examples.